Sources/Apollo/DefaultInterceptorProvider.swift

```diff
@@ -35,6 +35,7 @@ open class DefaultInterceptorProvider: InterceptorProvider {
         CacheReadInterceptor(store: self.store),
         NetworkFetchInterceptor(client: self.client),
         ResponseCodeInterceptor(),
+        MultipartResponseParsingInterceptor(),
```

Comment:
This is in the interceptor chain for all operation types. Right now it is applicable only to `Subscription` but supporting `@defer` will require it for `Query` too. There is a short-circuit to check for the `multipart/mixed` header and exit early if it's not present in the response.